src/lib/components/chat/ChatControls.svelte

```
@@ -2,7 +2,7 @@
 	import { SvelteFlowProvider } from '@xyflow/svelte';
 	import { slide } from 'svelte/transition';
 	import { Pane, PaneResizer } from 'paneforge';
-
+	import { NERDY_INTEGRATED } from '$lib/constants';
```

ESLint

[error] 'NERDY_INTEGRATED' is defined but never used.

(@typescript-eslint/no-unused-vars)